src/main/java/com/syncstream_dev/syncstream/model/user/User.java

```diff
@@ -0,0 +1,62 @@
+package com.syncstream_dev.syncstream.model.user;
+
+import java.time.LocalDateTime;
+
+
+public class User {
+    private UserInfo userInfo;
+    private String uid;     // User id
+    private String username;
+    private int messagesSent;
+    private int points;
+
+    // Register User Constructor
+    public User(String uid, String username, String token, String email, String password, String avatarUrl) {
+        // Todo: Database integration
+        this.userInfo = new UserInfo(token, email, password, LocalDateTime.now(), LocalDateTime.now(), avatarUrl);
+        this.uid = uid;
+        this.username = username;
+        this.messagesSent = 0;
+        this.points = 0;
+    }
+
+    // Login User Constructor
+    public User(String username, String password) {
+        throw new UnsupportedOperationException("Not implemented yet.");    // Todo: Requires database integration to load user info
```

Comment:
`// Todo:` on its own line so it does not look like documentation.

P.S. `TODO` in all caps for readability.

Reply:
ddea308b2335cfae4542d79046a8495ecf95e8e1 addresses this